js/runtime.go

```diff
@@ -123,22 +123,44 @@ func (r *Runtime) extractOptions(exports otto.Value, opts *lib.Options) error {
 	return nil
 }
 
-func (r *Runtime) loadFile(filename string) (otto.Value, error) {
-	// To protect against directory traversal, prevent loading of files outside the root (pwd) dir
-	path, err := filepath.Abs(filename)
+func (r *Runtime) loadJSCode(src *lib.SourceData) (otto.Value, error) {
```

Comment:
I'll change the name, then maybe you can reorganize as you see fit. I don't feel like I fully understand the differences between e.g. how loadLib() and loadFile() use load() - I'll probably just break things!